### DISCUSSION TITLE
Mission lightcurve data products

### DISCUSSION THREAD

**Discussion by sourestdeeds** [lightkurve/lightkurve#923] [Q&A] [answered]
I was wondering if there's a way of obtaining a list for all exoplanets in the MAST archive with available lightcurve files available for TESS? At the moment its alot of guesswork to query for dataproducts, there must be a way that I've overlooked on the MAST website itself.

**Comment by barentsen** [comment, marked as answer]
I'm not aware of such a list at this time!

Fortunately, it may be reasonable easy to create one by looping through a list of exoplanets (e.g. obtained from the NASA Exoplanet Archive) and calling `lightkurve.search_lightcurve` or `lightkurve.search_tesscut` for each target.

**Reply by sourestdeeds** [reply]
Good shout! I'll try this, if I'm successful ill let you know the results.

**Reply by sourestdeeds** [reply]
When performing a query on the MAST website itself for WASP-11 b I noticed there is a TESS lightcurve available. Using search_lightcurve returns no results however, even when I increase to radius to 100 arcseconds. Only 1000 arcseconds returns the result shown on the MAST archive. 

The target number of search_lightcurve matches that shown on MAST when I increase the search radius (641116633). I haven't verified if this actually corresponds to WASP-11 b yet, but it feels odd to have to increase my field of view to that extent for it to show.

Interestingly, it doesn't seem possible to search TIC id's on the MAST website itself either!

Its a similar situation for quite a few targets. I think I might have to make a translator to turn the TIC id's into reclassified names from the nasa exoplanet archive just to verify I'm definitely using the correct curves.

**Reply by barentsen** [reply]
Ah!  What's happening here is that WASP-11 b was not on the official TESS target list, but was serendipitously observed in the Full Frame Image data.  Light curve products for such FFI targets have only recently begun to be published by two pipelines (QLP & SPOC). You will need the very latest beta version of Lightkurve (v2.0b4) to access them, and specify `author="any"`.  For example:

<img width="977" alt="Screen Shot 2020-12-22 at 20 58 43" src="https://user-images.githubusercontent.com/817669/102961186-90345a80-4498-11eb-9be8-9e80d86b0546.png">

This will not work for targets in early TESS sectors just yet, because the ingestion of these new light curve products into MAST is still ongoing.  (The most recent sectors are being ingested first.)  Even then, these pipelines do not produce light curves for *every* star, but for a (very large) set of stars selected by some criteria (e.g. there is a brightness limit).

To know for certain whether a planet fell in the TESS pixel data, you can use `search_tesscut`.  For example:

<img width="627" alt="Screen Shot 2020-12-22 at 21 03 18" src="https://user-images.githubusercontent.com/817669/102961448-2799ad80-4499-11eb-8a3c-eb51b63fe1a3.png">

**Reply by sourestdeeds** [reply]
Interesting! For WASP-18 b MAST searches 0.2 degrees and returns 8 lightcurves for 3 different target_name(s). Are these also a result of later processing that they correspond to the same star? I double checked and sure enough, a search radius of 750 arcseconds with lightkurve returns all 8 results across 3 different target_names(s).

My worry is that in an oversight I had been accidentally using incorrect data for targets I had in mind!

Further to this, the DIAMENTE lightcurves have no TESS Sector metadata which is strange.
